### PR TITLE
Update Swedish translation

### LINKED ISSUE #1409: Zonemaster-Engine: Update translation to Swedish

### PR DESCRIPTION
**Pull request by mattias-p**
## Purpose

This PR updates the Swedish translation.

## Context

Fixes #1409.

## Changes

Translations in `sv.po` are updated to correspond to current develop branch.

## How to test this PR

N/A

### CODE REVIEW

**Review by matsduf — CHANGES_REQUESTED** [review]

File: share/sv.po [hunk 1]
```diff
@@ -1198,6 +1209,199 @@ msgstr ""
...
+"rekommenderade är att använda ett tomt salt. Hämtad från namnservrar "
+"\"{ns_list}\"."
+
+#. DNSSEC:DS03_INCONSISTENT_HASH_ALGO
+msgid ""
+"Inconsistent hash algorithm in NSEC3 in responses for the child zone from "
+"different name servers."
+msgstr ""
+"Inkonsistent hash-algoritm i NSEC3 i svar från olika namnservrar för "
+"barnzonen."
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po:1268]: I like "inkonsekvent" better too. I found it in couple more places. I'll fix those too.

**Review by matsduf — COMMENTED** [review]
There is a risk that the changes to the non-sv PO files will create conflicts with other translators. It might be better to do that reformatting at the end after translation?

File: share/sv.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po]: Fixed.

File: share/sv.po [hunk 2]
```diff
@@ -1198,6 +1209,199 @@ msgstr ""
...
+
+#. DNSSEC:DS03_INCONSISTENT_ITERATION
+msgid ""
+"Inconsistent NSEC3 iteration value in responses for the child zone from "
+"different name servers."
+msgstr ""
+"Inkonsistent NSEC3-iterationsvärde i svar från olika namnservrar för "
+"barnzonen."
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po]: Fixed.

File: share/sv.po [hunk 3]
```diff
@@ -1198,6 +1209,199 @@ msgstr ""
...
+
+#. DNSSEC:DS03_INCONSISTENT_NSEC3_FLAGS
+msgid ""
+"Inconsistent NSEC3 flag list in responses for the child zone from different "
+"name servers."
+msgstr ""
+"Inkonsistent NSEC3-flagglistevärde i svar från olika namnservrar för "
+"barnzonen."
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po]: Fixed.

File: share/sv.po [hunk 4]
```diff
@@ -1198,6 +1209,199 @@ msgstr ""
...
+
+#. DNSSEC:DS03_INCONSISTENT_SALT_LENGTH
+msgid ""
+"Inconsistent salt length in NSEC3 in responses for the child zone from "
+"different name servers."
+msgstr ""
+"Inkonsistent saltlängd i NSEC3 i svar från olika namnservrar för barnzonen."
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po]: Fixed.

File: share/sv.po [hunk 5]
```diff
@@ -1198,6 +1209,199 @@ msgstr ""
...
+
+#. DNSSEC:DS03_ERR_MULT_NSEC3
+#. DNSSEC:DS10_ERR_MULT_NSEC3
+#, perl-brace-format
+msgid ""
+"Multiple NSEC3 records when one is expected. Fetched from name servers "
+"\"{ns_list}\"."
+msgstr ""
+"Flera NSEC3-poster hittades när endast en förväntades. Hämtad från "
+"namnservrarna \"{ns_list}\"."
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po:1229]: I agree. I couldn't come up with anything better either.

**Comment by mattias-p** [comment]
@matsduf I've resolved all comments and fixed CI. Please review again.

**Review by matsduf — APPROVED** [review]

**Pull request merged**